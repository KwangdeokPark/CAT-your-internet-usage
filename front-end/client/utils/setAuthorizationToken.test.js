import axios from 'axios';
import setAuthorizationToken from './setAuthorizationToken';

describe('sAT', () => {
  it('should do what i like', () =>{
    expect(setAuthorizationToken('orange')).toMatchSnapshot();
  });
})
