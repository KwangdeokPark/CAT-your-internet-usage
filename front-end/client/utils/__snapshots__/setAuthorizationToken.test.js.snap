// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sAT should do what i like 1`] = `undefined`;
